definitions/npm/winston_v3.x.x/flow_v0.34.x-/winston_v3.x.x.js

```diff
@@ -84,9 +85,26 @@ declare class $winstonContainer<T> {
   constructor(config?: $winstonLoggerConfig<T>): $winstonContainer<T>;
   add(loggerId: string, config?: $winstonLoggerConfig<T>): $winstonLogger<T>;
   get(loggerId: string): $winstonLogger<T>;
+  has(loggerId: string): boolean;
 }
 
 declare module "winston" {
+  declare export type $WinstonLevels = $winstonLevels;
+  declare export type $WinstonNpmLogLevels = $winstonNpmLogLevels;
+  declare export type $WinstonInfo<T: $WinstonLevels > = $winstonInfo<T>;
+  declare export type $WinstonFormat = $winstonFormat;
+  declare export type $WinstonFileTransportConfig<T: $WinstonLevels> = $winstonFileTransportConfig<T>;
+  declare export type $WinstonTransport = typeof $winstonTransport;
+  declare export type $WinstonFileTransport<T: $WinstonLevels> = $winstonFileTransport<T>;
+  declare export type $WinstonConsoleTransportConfig<T: $WinstonLevels> = $winstonConsoleTransportConfig<T>;
+  declare export type $WinstonConsoleTransport<T: $WinstonLevels> = $winstonConsoleTransport<T>;
+  declare export type $WinstonLoggerConfig<T: $WinstonLevels> = $winstonLoggerConfig<T>;
+  declare export type $WinstonLogger<T: $WinstonLevels> = $winstonLogger<T>;
+  declare export type $WinstonConfigSubModule = $winstonConfigSubModule;
+  declare export type $WinstonFormatSubModule = $winstonFormatSubModule;
+  declare export type $WinstonDefaultLogger = $winstonDefaultLogger;
+  declare export type $WinstonContainer<T: $WinstonLevels> = $winstonContainer<T>;
```

Comment:
The `$winston` prefix was used because those types are global. 
In a module scope this is not necessary.  
I would rather remove them personally.  `Levels`, `NpmLogLevel`, `Format`, etc.

Reply:
Prefix removed.